gnc/ctl/src/ctl.cc

```diff
@@ -510,7 +510,7 @@ bool Ctl::Step(void) {
       NODELET_DEBUG_STREAM_THROTTLE(10, "GNC step waiting for inertia");
       return false;
     }
-    if (!control_enabled_) {
+    if (!flight_enabled_ || !control_enabled_) {
```

Comment:
please double check this condition to make sure it's sane

Reply:
Looks right to me!

Reply:
Looks good to me! `control_enabled_` should stay set to the value set by the service this way.